<?xml version="1.0"?>
<launch>
  <arg name="img" default="/image_raw"/>
  <arg name="machine" default="this_machine"/>
  <arg name="rgbOrFlow" default="rgb"/>
  <arg name="split" default="1"/>
  <arg name="cfw" default="50"/>

  <include file="$(find caffe_tsn_ros)/launch/common.launch">
    <arg name="img" default="$(arg img)"/>
    <arg name="machine" default="$(arg machine)"/>
    <arg name="rgbOrFlow" default="$(arg rgbOrFlow)"/>
    <arg name="split" default="$(arg split)"/>
    <arg name="cfw" default="$(arg cfw)"/>
    <arg name="nodetype" value="class_tsn.py"/>
  </include>
</launch>
